plugin/trino-bigquery/README.md

```diff
@@ -22,9 +22,16 @@ You can follow the steps below to be able to run the integration tests locally.
   **BigQuery Admin** role assigned.
 * Get the base64 encoded text of the service account credentials file using `base64
   /path/to/service_account_credentials.json`.
+* Create a new BigQuery `ClOUD_RESOURCE` connection and grant the connection service account GCS permissions.
```

Comment:
small typo `CLOUD_RESOURCE` instead of `ClOUD_RESOURCE` (capital `L`)

Reply:
Thank you. Fixed.